spec/support/features/idv_helper.rb

```diff
@@ -96,4 +96,29 @@ def complete_idv_profile_ok(user, password = user_password)
     fill_in 'Password', with: password
     click_submit_default
   end
+
+  def visit_idp_from_sp_with_loa3(state: SecureRandom.hex, client_id:, nonce:)
```

Comment:
Good points. I renamed the methods here: https://github.com/18F/identity-idp/pull/1581/commits/abcd42cc78ab787d2658d9c4c804e2f48d769bf5